### PR TITLE
local manager test

### PR DESCRIPTION
**Pull request by giggsoff**
Local profile, local profile server and its token are now in configurable params;
Implementation of the local profile server and test for profile logic.

Signed-off-by: Petr Fedchenkov <giggsoff@gmail.com>

### CODE REVIEW

File: tests/eclient/testdata/profile.txt [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+exec chmod 600 {{EdenConfig "eden.tests"}}/eclient/image/cert/id_rsa
+
+# Starting of reboot detector with a 1 reboot limit
+! test eden.reboot.test -test.v -timewait 40m -reboot=0 -count=1 &
+
+# Define local-manager and two apps in different profiles
+eden pod deploy -n local-manager --memory=512MB docker://itmoeve/eclient:0.7 -p 2223:22
+eden pod deploy -n app-profile-1 --memory=512MB docker://itmoeve/eclient:0.7 --profile=profile-1
+eden pod deploy -n app-profile-2 --memory=512MB docker://itmoeve/eclient:0.7 --profile=profile-1
+
```
Comment (eriknordmark) [inline tests/eclient/testdata/profile.txt:19]: Somewhere we need to have apps which can run in two configs.I think that means an app instance can be in more than one profile. Thus a third app instance  which is in profile-1 and profile-2 (and then it you switch to "profile-3" they both halt.

File: tests/eclient/testdata/profile.txt [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+exec chmod 600 {{EdenConfig "eden.tests"}}/eclient/image/cert/id_rsa
+
+# Starting of reboot detector with a 1 reboot limit
+! test eden.reboot.test -test.v -timewait 40m -reboot=0 -count=1 &
+
+# Define local-manager and two apps in different profiles
+eden pod deploy -n local-manager --memory=512MB docker://itmoeve/eclient:0.7 -p 2223:22
+eden pod deploy -n app-profile-1 --memory=512MB docker://itmoeve/eclient:0.7 --profile=profile-1
+eden pod deploy -n app-profile-2 --memory=512MB docker://itmoeve/eclient:0.7 --profile=profile-1
```
[the remark this answers is not in the document]
Comment (giggsoff) [reply tests/eclient/testdata/profile.txt:18]: Right, sorry, `profile-2` here

File: tests/eclient/testdata/profile.txt (hunk 1 above)
Comment (giggsoff) [reply tests/eclient/testdata/profile.txt:19]: Ok, will add it.

File: tests/eclient/testdata/profile.txt [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+# Wait for ssh access
+exec -t 10m bash wait_ssh.sh 2223
+
+# Define local_manager field in device config
+# TBD: set device option, obtain IP address
+eden controller edge-node update --config local_manager=10.11.12.2:80
+
+# use profile-1 variable
+exec -t 1m bash profile-1.sh
```
[the remark this answers is not in the document]
Comment (giggsoff) [reply tests/eclient/testdata/profile.txt:31]: And another question: should we mark app instance local_manager in any way? I mean, we should later check manager_uuid, how we plan to determinate, is app instance manager or not?

**Pull request merged**